### PR TITLE
Migrated `IniFileConfigStore` to `@configu/integrations`

### LINKED ISSUE #516: Migrate `IniFileConfigStore` to `@configu/integrations`
### Suggestion

Migrate the `IniFileConfigStore` from its current setup to the new `@configu/integrations` module as part of the major restructuring. The goal is to move all third-party integrations, including stores, expressions, and libraries, to `@configu/integrations`.

### Motivation

This migration is part of a larger effort to better organize and modularize the product. Moving the `IniFileConfigStore` will align with the new structure, making the system cleaner and more maintainable.

### Context

You can refer to the already migrated stores for guidance:  
[Migrated stores in integrations](https://github.com/configu/configu/tree/main/packages/integrations/stores).

Base stores implementations from `@configu/ts` can be found here:  
[Base stores in TS](https://github.com/configu/configu/tree/52cee9c41fb03addc4c0983028e37df42945f5b7/packages/ts/src/stores).

The old node-based stores were previously in `@configu/node`:  
[Node stores](https://github.com/configu/configu/tree/52cee9c41fb03addc4c0983028e37df42945f5b7/packages/node/src/stores).

The old browser-based stores were previously in `@configu/browser`:  
[Browser stores](https://github.com/configu/configu/tree/52cee9c41fb03addc4c0983028e37df42945f5b7/packages/browser/src/stores).

### PR DESCRIPTION
**Pull request by SanchitMahajan236**
Closes #516

### CODE REVIEW

File: packages/integrations/stores/ini-file/src/ini.ts [hunk 1]
```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash';
+import ini from 'ini';
+import { type Config } from '@configu/ts';
```
Comment (RichardAkman) [inline packages/integrations/stores/ini-file/src/ini.ts:3]: Please replace the use of `@configu/ts` with `@configu/sdk`, make sure to remove the dependency from the `package.json` file.

File: packages/integrations/stores/ini-file/src/ini.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import _ from 'lodash';
```
Comment (RichardAkman) [inline packages/integrations/stores/ini-file/src/ini.ts:1]: Replace with `lodash-es` and make sure it's a dependency in `package.json`.

File: packages/integrations/stores/ini-file/package.json [hunk 3]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@configu-integrations/ini",
```
Comment (RichardAkman) [inline packages/integrations/stores/ini-file/package.json:2]:
```suggestion
  "name": "@configu-integrations/ini-file",
```

File: packages/integrations/stores/ini-file/package.json
Comment (RichardAkman) [inline packages/integrations/stores/ini-file/package.json:1]: Please also include a `README.md` file and use the same file structure from any of the PRs related to #600.

**Review by RichardAkman — CHANGES_REQUESTED** [review]
Great work ⭐ , please apply my comments before we can proceed

**Comment by RichardAkman** [comment]
@SanchitMahajan236 please let us know if you're still working on this PR, if not let us know and we will make the issue available to other contributors 🙏🏻

**Comment by SanchitMahajan236** [comment]
Hi @RichardAkman , I have done the changes to the code as mentioned by you and working on the readme file.

**Comment by SanchitMahajan236** [comment]
Hi @RichardAkman , I have merged the updates requested.

**Review by RichardAkman — APPROVED** [review]
Amazing work! ⭐ 🚀 

Made a few minor adjustments to align this store with other stores

**Comment by pelegpor** [comment]
Hi @SanchitMahajan236 ,

Here are the details for claiming your Hacktoberfest prizes! 🎉 Check out [#465  (comment)](https://github.com/configu/configu/issues/465#issuecomment-2498550259) for more information or simply fill out the form here: [Claim Your Swag](https://share-eu1.hsforms.com/2SVc5VKY3Thi04Bvjh17WKw2e62o0).  

Remember, you only need to fill out the form once for all your PRs. 😊

**Pull request merged**